ungrib/Variable_Tables/README

```diff
@@ -21,6 +21,8 @@ Vtable.ECMWF_sigma         ECMWF sigma-level output
 Vtable.ERA-interim.ml           ERA interim model level
 Vtable.ERA-interim.pl           ERA interim pressure level output
 
+Vtable.ERA5                ERA5 single level and pressure level data from the Climate Data Store 
```

Comment:
@JoshLovesFun Do you mean 'sigma' level?

Reply:
@weiwangncar In this case, it is actually pressure levels. Because the specific dataset that this Vtable is targeting is the ERA5 data in the climate data store that has already been regridded to pressure levels from the native hybrid-sigma levels. So thinking about it more, this Vtable should be called "Vtable.ERA5.pl".

If I work with the native ERA5 model data in the future, I could make a "Vtable.ERA5.native"

I think the native ERA5 data is a bit more difficult to download and use, but I haven't tried it. I think most user's would get the pressure level data.